### PR TITLE
feat: use biome linter

### PR DESCRIPTION
**Pull request by RafaelGSS**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
-  "dependencies": {
-    "piscina": "^4.8.0"
-  }
+	"name": "bench-node",
+	"version": "0.5.0",
+	"description": "",
+	"main": "lib/index.js",
+	"scripts": {
+		"test": "node --test --allow-natives-syntax",
+		"lint": "biome lint .",
```
Comment (H4ad) [inline package.json:8]: From what I remember, it's a good idea to keep separated the lint for CI and lint for local machine, so create a lint for ci and call: `biome ci .`

File: biome.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (H4ad) [inline biome.json:1]: I already used biome once, I used the following config: https://github.com/kibertoad/unzipomatic/blob/main/biome.json

**Review by H4ad — APPROVED** [review]
Besides adding the `lint:ci`, looks good!

File: biome.json (hunk 2 above)
Comment (RafaelGSS) [reply biome.json:1]: I'm using the same configuration we are using on `amaro` and `nodejs-loaders`.

**Pull request merged**